api/helpers/handlebars-helpers.js

```diff
@@ -170,13 +171,24 @@ module.exports = {
     // string like "value" which represents the key
 
     let key;
-    if(article[name].key) {
-      key = article[name].key
+    if (article[name].key) {
+      key = article[name].key;
     } else {
       key = article[name];
     }
 
+<<<<<<< HEAD
```

Comment:
you only need to keep this line to resolve this conflict: 
`return i18n(`name:${name}-key:${key}`, context);`

Reply:
Oh, I resolved the wrong way?  Thanks!

Reply:
Oh, no, I see what you mean. I missed a conflict. Even worse!